.changeset/angry-otters-joke.md

```diff
@@ -0,0 +1,6 @@
+---
+"@latticexyz/common": patch
+"@latticexyz/explorer": patch
+---
+
+Added Pyrope Testnet chain configuration and support for Worlds Explorer.
```

Comment:
```suggestion
Added Pyrope chain.
```

this will show up contextually in each changeset

alternatively, could split them into separate changesets

Reply:
yeah makes sense, will split adding pyrope chain to common package as a separate PR